mmv1/third_party/terraform/website/docs/r/compute_region_instance_group_manager.html.markdown

```diff
@@ -141,10 +150,13 @@ The following arguments are supported:
 * `auto_healing_policies` - (Optional) The autohealing policies for this managed instance
 group. You can specify only one value. Structure is documented below. For more information, see the [official documentation](https://cloud.google.com/compute/docs/instance-groups/creating-groups-of-managed-instances#monitoring_groups).
 
+* `all_instances_config` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
+  Properties to set on all instances in the group. After setting
+  allInstancesConfig on the group, you must update the group's instances to
+  apply the configuration.
```

Comment:
missing stateful disk block and nested subfield explanation

Reply:
ditto, this PR is about all_instances_config. I would prefer to merge it without additional changes related to different feature documentation. WDYT about fixing  this in separate PR?